src/main.js

```diff
@@ -281,12 +281,13 @@ define('peaks', [
       get: function () {
         var self = this;
 
-        function addSegment (startTime, endTime, editable, color, labelText) {
+        function addSegment (startTime, endTime, editable, color, labelText, id) {
```

Comment:
The number of parameters is becoming unwieldy, so I'd prefer we address issue #6 before (or as part of) making this change.